### PR TITLE
DOC/PERF: changes to make within README

### PR DESCRIPTION
**Pull request by govarguz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ QUICKINSTALL:
 ```
 # cd espressopp
 # cmake -DEXTERNAL_BOOST=OFF -DEXTERNAL_MPI4PY=OFF .
-# make -j -l$(nproc)
+# make -j -l$(nproc-2)
```
Comment (junghans) [inline README.md]: In bash:
```
$ make -j -l$(nproc-2)
bash: nproc-2: command not found
make: *** No targets specified and no makefile found.  Stop.
```

**Comment by govarguz** [comment]
thnx! fixed the typo

File: README.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ QUICKINSTALL:
 ```
 # cd espressopp
 # cmake -DEXTERNAL_BOOST=OFF -DEXTERNAL_MPI4PY=OFF .
-# make -j -l$(nproc)
+# make -j -l$(nproc)-2
```
Comment (junghans) [inline README.md]: So how would that work for 1 and 2 core machines?

Comment (MrTheodor) [inline README.md]: What is $(nproc) ?

$ echo $nproc

Does not show anything on my machine...

Comment (junghans) [inline README.md]: I am still not convinced of the point of this change, using load=nproc-2 is as arbitrary as load=nproc.

Plus, when `make` parses the option using `atof()`, `atof(x-2)` becomes `atof(x)`, which effectively ignores the `-2`.

Comment (junghans) [inline README.md]: in bash:
```
make -j -l$(($(nproc)-2))
```
would work.

**Comment by niktre** [comment]
One of the reasons to provide a smaller number than nproc consists in a VERY slow response of a machine upon compilation. I cannot even move the pointer on my mac and my desktop

Von meinem iPhone gesendet

**Comment by govarguz** [comment]
@niktre exactly, that was the reason I would modify the bash line in a few moments!

**Comment by niktre** [comment]
maybe it is better to write:

`# make [for multicore CPUs make -j #threads]`

?

**Comment by junghans** [comment]
@niktre  Yeah, I think that is better!

Usually `make -j<cores+1>` is what I have seen used.

**Comment by govarguz** [comment]
Combined all suggestions for changes to make @ README.md

**Pull request merged**